docs/UserGuide.md

```diff
@@ -280,21 +280,23 @@ Format: `elist`
 
 Finds events whose name contain any of the given keywords.
 
-Format: `efind n/NAME…​`
+Format: `efind n/EVENT_NAME [n/MORE_EVENT_NAME]... ​`
```

Comment:
not sure if we should change to `EVENT_NAME` coz our error message for efind only shows `NAME`

Reply:
hmmm